File: Python/interactions/select_dropdown.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://example.com")
    page.select_option("select#country", "us")  # value attribute
    print(page.content())
    browser.close()

File: NodeJS/interactions/select_dropdown.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  await page.selectOption('select#country', 'us');
  console.log(await page.content());
  await browser.close();
})();
